lib/vagrant-openshift/action.rb

```diff
@@ -204,6 +210,30 @@ def self.install_rhc(options)
         end
       end
 
+      def self.bootstrap_openshift3(options)
```

Comment:
Sorry I didn't notice this before.  But this should really start from the steps after inst.  And to create a bootstrap image you launch instance and run bootstrap.